Fix crash for too large struct array indicies (#5926)
ethereum/solidity: Changelog.md

Bugfixes:
 * Code generator: Defensively pad allocation of creationCode and runtimeCode to multiples of 32 bytes.
 * Parser: Disallow empty import statements.

>  * Type Checker: Fix internal error when a struct array index is not an unsigned.

Reviewer: ```suggestion
 * Type Checker: Fix internal error when a struct array index is not an unsigned integer.
```

Reviewer: or
```suggestion
 * Type Checker: Fix internal error when a struct array index does not fit into an uint256.
```

Reviewer 2: "a uint" :)

Author: Mind you all, I just copied this change log entry from https://github.com/ethereum/solidity/pull/5195/files#diff-9b8fab691c00b9e5380b19ce882f3271R128
and added "struct" :P

>  * Type Checker: Fix internal error when a struct array index does not fit into an uint256.

Reviewer 2: ```suggestion
 * Type Checker: Fix internal error when a struct array index does not fit into a uint256.
```
